### PR TITLE
2087 new SecondOrder

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
Creates a new SecondOrder block in order to allow a functioning simulation under Dynawo. Block may be deleted if the problem the MSL component is solved.

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+block SecondOrder
+  import Modelica;
+  import Dynawo.Types;
+
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Real D "Damping";
+  parameter Real K = 1 "Gain";
+  parameter Real W "Angular frequency";
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo:23]: w without a capital letter for frequency. + units

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  annotation(preferredView = "diagram",
+    Documentation(info="<html>
+<p>
+This blocks defines the transfer function between the input u and
+the output y as <em>second order</em> system:
+</p>
+<pre>
+                           k
+   y = ---------------------------------------- * u
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo:84]: Use the same parameters here as in the model

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt [hunk 3]
```diff
@@ -22,6 +22,7 @@ set(MODEL_FILES
   IntegratorOrFirstOrder.mo
   LimitedIntegrator.mo
   LimitedLeadLag.mo
+  Limiter.mo
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt:25]: Have you rebased ? This should not appear as already here

**Review by cossartque — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt (hunk 3 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt:25]: I have completed this file manually. I shouldn't ?

Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt:25]: You should rebase your branch on master. This file is already in master, it is not a new model so it should not appear.

Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt:25]: Of course.

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo (hunk 2 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo:84]: k replaced by K

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo (hunk 1 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo:23]: w and Angular frequency in 1/s

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+block SecondOrder
+  import Modelica;
+  import Dynawo.Types;
+
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Types.PerUnit D "Damping";
+  parameter Types.PerUnit K = 1 "Gain";
+  parameter Types.Frequency w "Angular frequency in 1/s";
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo:23]: why not Hz ?

Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo]: Indeed.

**Review by cossartque — APPROVED** [review]

**Pull request merged**